### PR TITLE
i18n(tr): update `components.md` file

### PR DESCRIPTION
**Pull request by erkanaltinors**
What kind of changes does this PR include?

Updates of Starlight docs site content

Description

This pull request updates translated in Turkish `components.md` file in project.

### CODE REVIEW

File: docs/src/content/docs/tr/guides/components.mdx [hunk 1]
```diff
@@ -65,18 +65,66 @@ Her `<TabItem>` bileşenini kullanıcılara göstermek için `label` sahibi olma
 import { Tabs, TabItem } from '@astrojs/starlight/components';
 
 <Tabs>
-	<TabItem label="Yıldızlar">Sirius, Vega, Betelgeuse</TabItem>
-	<TabItem label="Uydular">Io, Europa, Ganymede</TabItem>
+	<TabItem label="Yıldızlar" icon="star">Sirius, Vega, Betelgeuse</TabItem>
```
Comment (HiDeoo) [inline docs/src/content/docs/tr/guides/components.mdx:68]: I cannot leave a comment for the sentences above the example but it looks like this paragraph contains a few issues:

- The "You can display a tabbed interface using the `<Tabs>` and `<TabItem>` components." is in English.
- It's missing the `icon` attribute description and link

Would you mind taking a look at it and updating it?

File: docs/src/content/docs/tr/guides/components.mdx [hunk 2]
```diff
@@ -164,23 +212,220 @@ import { LinkCard } from '@astrojs/starlight/components';
 	<LinkCard title="Bileşenler" href="/tr/guides/components/" />
 </CardGrid>
 
+### Ara bölümler
+
+Ara bölümler (ayrıca "uyarı" ve "açıklama balonu" olarak da bilinir), sayfanın ana içeriği ile birlikte ikincil bilgi göstermek için kullanışlıdır.
```
Comment (HiDeoo) [inline docs/src/content/docs/tr/guides/components.mdx:217]:
```suggestion
Ara bölümler (ayrıca "uyarı" ve "açıklama balonu" olarak da bilinir), sayfanın ana içeriği ile birlikte ikincil bilgi göstermek için kullanışlıdır.

```

To match the English version

File: docs/src/content/docs/tr/guides/components.mdx [hunk 3]
```diff
@@ -164,23 +212,220 @@ import { LinkCard } from '@astrojs/starlight/components';
...
+
+import { Aside } from '@astrojs/starlight/components';
+
+<Aside>Özel başlığı olmayan bir varsayılan ara bölüm</Aside>
+
+<Aside type="caution" title="Dikkat Et!">
+	Özel bir başlık *ile* uyarı ara bölümü.
+</Aside>
+
+<Aside type="tip">
```
Comment (HiDeoo) [inline docs/src/content/docs/tr/guides/components.mdx:231]:
```suggestion
<Aside type="tip">

```

Just to match the English version too

File: docs/src/content/docs/tr/guides/components.mdx [hunk 4]
```diff
@@ -164,23 +212,220 @@ import { LinkCard } from '@astrojs/starlight/components';
...
+```js
+// Örneğin bir kod parçacığı.
+```
+
+</Aside>
+
+<Aside type="danger">Şifreni kimseye verme.</Aside>
+
+Starlight, ayrıca `<Aside>` bileşenine alternatif olarak Markdown ve MDX içerisinde ara bölüm oluşturmak için özel bir sözdizimi sağlar.
+Starlight also provides a custom syntax for rendering asides in Markdown and MDX as an alternative to the `<Aside>` component.
```
Comment (HiDeoo) [inline docs/src/content/docs/tr/guides/components.mdx:266]:
```suggestion
```

Probably something missed

File: docs/src/content/docs/tr/guides/components.mdx [hunk 5]
```diff
@@ -164,23 +212,220 @@ import { LinkCard } from '@astrojs/starlight/components';
...
+  - …
+- pages/
+
+</FileTree>
+
+### Adımlar
+
+`<Steps>` bileşenini numaralanmış görevler listesini biçimlendirmek için kullanın.
+
+Her aşamasının açıkça belirtilmeye ihtiyaç duyan çok karmaşık adım adım rehberleri oluşturmak için kullanışlıdır.
```
Comment (HiDeoo) [inline docs/src/content/docs/tr/guides/components.mdx:368]:
```suggestion
`<Steps>` bileşenini numaralanmış görevler listesini biçimlendirmek için kullanın.
Her aşamasının açıkça belirtilmeye ihtiyaç duyan çok karmaşık adım adım rehberleri oluşturmak için kullanışlıdır.
```

To match the english version

File: docs/src/content/docs/tr/guides/components.mdx [hunk 6]
```diff
@@ -164,23 +212,220 @@ import { LinkCard } from '@astrojs/starlight/components';
...
+
+Standart Markdown sıralanmış listesini `<Steps>` bileşeni ile sarmalayın.
+
+`<Steps>` bileşeni içerisinde tüm olağan Markdown sözdizimi uygulanabilir.
```
Comment (HiDeoo) [inline docs/src/content/docs/tr/guides/components.mdx:372]:
```suggestion
Standart Markdown sıralanmış listesini `<Steps>` bileşeni ile sarmalayın.
`<Steps>` bileşeni içerisinde tüm olağan Markdown sözdizimi uygulanabilir.
```

Same as above

File: docs/src/content/docs/tr/guides/components.mdx [hunk 7]
```diff
@@ -164,23 +212,220 @@ import { LinkCard } from '@astrojs/starlight/components';
...
+
 ### İkon
 
 import { Icon } from '@astrojs/starlight/components';
 import IconsList from '~/components/icons-list.astro';
 
 Starlight, içeriğinizde görünmesi için `<Icon>` bileşenini kullanarak bir grup ortak ikonları kullanımınıza sunar.
-Her `<Icon>` [`name`](#tüm-i̇konlar) değerine ihtiyaç duyar ve isteğe bağlı olarak `label`, `size`, ve `color` özellikleri eklenebilir.
+Her `<Icon>` [`name`](#tüm-i̇konlar) değerine ihtiyaç duyar ve isteğe bağlı olarak `label` eklenebilir.
+`size` ve `color` nitelikleri ikonun görünümünü CSS birimleri ve renk değerleri ile ayarlamak için kullanılabilir.
```
Comment (HiDeoo) [inline docs/src/content/docs/tr/guides/components.mdx:414]:
```suggestion

Her `<Icon>` [`name`](#tüm-i̇konlar) değerine ihtiyaç duyar ve isteğe bağlı olarak `label` eklenebilir.
`size` ve `color` nitelikleri ikonun görünümünü CSS birimleri ve renk değerleri ile ayarlamak için kullanılabilir.
```

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). I left a few tiny  suggestions but this is looking pretty great :thumbsup:

**Comment by erkanaltinors** [comment]
@HiDeoo I've applied all suggessted changes in last commit.

**Review by HiDeoo — APPROVED** [review]
> @HiDeoo I've applied all suggessted changes in last commit.

Perfect and super fast, thanks a lot for another great translation :raised_hands:

**Pull request merged**